waterloop-site/src/App.tsx

```diff
@@ -100,6 +101,7 @@ class App extends React.Component<{}, State> {
             <Route path="/posting/:id">
               <JobPostingPage />
             </Route>
+            <Route component={ErrorPage} />
```

Comment:
Great! But just one more thing, we want to redirect to a /404 URL. Meaning that, if the URL is 

teamwaterloop.ca/asdfdsalfkasdf, it will redirect to teamwaterloop.ca/404, and show this page.

Reply:
If you're still confused about that, check the documentation: https://reactrouter.com/web/api/Redirect